packages/components/Tabs/src/Tabs.types.ts

```diff
@@ -0,0 +1,117 @@
+import * as React from 'react';
+import { ITextProps } from '@fluentui-react-native/text';
+import type { IViewWin32Props } from '@office-iss/react-native-win32';
+import { IRenderData } from '@uifabricshared/foundation-composable';
+import { IForegroundColorTokens, FontTokens, IBackgroundColorTokens } from '@fluentui-react-native/tokens';
+import { FocusZoneProps } from '@fluentui-react-native/focus-zone';
+import { View, ViewProps } from 'react-native';
+
+export const tabsName = 'Tabs';
+
+export interface ITabsContext {
+  /*
+   ** The currently selected TabsItem's key
+   */
+  selectedKey: string | null;
+
+  /*
+   ** Index of currently selected key
+   */
+
+  getTabId?: (key: string, index: number) => void;
+
+  /*
+   ** Updates the selected tabsItem and calls the client’s onTabsClick callback
+   */
+  onTabsClick?: (key: string) => void;
+
+  /*
+   ** Updates the selected tabsItem's ref to set as the default tabbable element
+   */
+  updateSelectedTabsItemRef?: (ref: React.RefObject<any>) => void;
+
+  /*
+   ** Array of tabsItem keys in the group
+   */
+  tabsItemKeys?: string[];
+
+  /*
+   ** A Map to for a TabItems corresponding view
+   */
+  views?: Map<string, React.ReactNode[]> | null;
+}
+
+export interface TabsInfo {
+  headersOnly?: boolean;
+  label?: boolean,
+}
+
+export interface TabsState {
+  context: ITabsContext;
+  info: TabsInfo;
+}
+
+export interface TabsProps extends Pick<FocusZoneProps, 'isCircularNavigation'> {
+  /*
+   ** Descriptive label for the Tabs. This will be displayed as the title of the Tabs to the user
+   */
+  label?: string;
+
+  /*
+   ** The key of the TabsItem that will initially be selected
+   */
+  defaultSelectedKey?: string;
+
+  /*
+   **  OPTIONAL: An aria label for narrator. If not provided, it will be set to the label of the Tabs
+   */
+  ariaLabel?: string;
+
+  /*
+   ** The key of the selected option. If you provide this, you must maintain selection state by observing
+   ** onTabsClick events and passing a new value in when changed. This overrides defaultSelectedKey
+   ** and makes the Tabs a controlled component.
+   */
+  selectedKey?: string;
+
+  /*
+   ** Callback for receiving a notification when the choice has been changed
+   */
+  onTabsClick?: (key: string) => void;
+
+  /*
+   ** Callback to customize how IDs are generated for each tab header.
+   ** Useful if you're rendering content outside and need to connect aria-labelledby.
+   */
+  getTabId?: (key: string, index: number) => string;
+
+  /*
+   ** Sets whether to only render the header
+   */
+  headersOnly?: boolean;
+
+  /*
+   ** Sets whether to only render the header
+   */
+  componentRef?: React.RefObject<View>;
+
+  testID?: string;
+}
+
+export interface TabsTokens extends IForegroundColorTokens, FontTokens, IBackgroundColorTokens {}
+
+export interface TabsSlotProps {
+  root: React.PropsWithRef<IViewWin32Props>;
+  label: ITextProps;
+  container: ViewProps & FocusZoneProps;
```

Comment:
Could you explain why ViewProps and FocusZoneProps are listed here for container?

Reply:
Because we need to style the container that holds the TabsItems with flex. If we just have FocusZoneProps TypeScript complains about styling the FocusZone.

Reply:
Maybe you could create another slot to handle the style/orientation of the container holding the children of Tabs? Named "stack" or "inner" or something else and have it be a View or Stack? Just throwing out an idea. Not sure if it's preferable to the solution you already have.

Reply:
A Krystal said above, I would make a separate Slot called stack or something. Make this a View and have that be a child to the FocusZone. 

I'm also thinking about the future of the control. If we end up allowing the user to change the orientation (vertical vs. horizontal), and other styling things, it makes more sense to separate this into separate slots.